server/controllers/admin.py

```diff
@@ -290,7 +302,7 @@ def autograde_backup(bid):
     return redirect(url_for('.grading', bid=bid))
 
 @admin.route("/versions/<name>", methods=['GET', 'POST'])
-@is_staff()
+@is_admin()
```

Comment:
I did a pass and confirmed that `is_staff()` is used appropriately but I could see a strong case for renaming it so that future maintainers are less confused.